stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerDefaultCenterContent.kt

```diff
@@ -38,18 +45,13 @@ public class MessageComposerDefaultCenterContent : FrameLayout, MessageComposerC
     /**
      * Callback invoked when clear button was clicked.
      */
-    public var onClearButtonClicked: () -> Unit = {}
+    public var onClearButtonClick: () -> Unit = {}
```

Comment:
So there are two way to name listeners

- For a thing that already happened
- For a thing that will happen

Here - we're exposing some handle when the button is _clicked_. So when something already happened. There are situation where we want actions that _will_ happen.

E.g. `onSendMessage` might be better than `onMessageSent`, because it actually didn't send the message yet - we exposed an interceptor to the send action and the user might define logic that doesn't allow us to send the message in the end.

Does this make sense? Due to this - I feel like `onClearButtonClicked` is okay, but I feel something like `onClearInput` might be better (as the user actually intercepts the clear action).